kanopi/Engineering-Best-Practices
# Feature circle

## Description
- Update front page table of contents to use flexbox instead of floats.
- Add docs about CircleCI.
- Add Readme docs about how to do certain repetitive tasks to offload from 🧠

## Affected URL
http://127.0.0.1:4000/Engineering-Best-Practices/circleci/

## Related Tickets
no ticket

## Steps to Validate
1. Review front page -- check responsive as well -- new flexbox layout
2. Review CircleCI page for correctitudiness
3. Review Tools page to ensure link to CircleCI page is still there and doing its thing
4. Review Readme update to make sure it makes sense and is not gibberish
